### PR TITLE
Allow for multiple chains rpc

### PR DESCRIPTION
**Pull request by marc-aurele-besner**
## Allow for multiple chains rpc

- Allow Astral to switch between multiple chain rpc for rpc data and transactions

### CODE REVIEW

File: client/src/Operator/components/OperatorStake.tsx [hunk 1]
```diff
@@ -142,8 +142,8 @@ const OperatorStake = () => {
         const hash = await api[selectedChain.urls.page].tx.domains
           .registerOperator(values.domainId, values.amountToStake * 10 ** tokenDecimals, {
             signingKey: values.signingKey,
-            minimumNominatorStake: values.minimumNominatorStake * 10 ** tokenDecimals,
-            nominationTax: values.nominatorTax,
+            minimumNominatorStake: (values.minimumNominatorStake * 10 ** tokenDecimals).toString(),
```
Comment (jfrank-summit) [inline client/src/Operator/components/OperatorStake.tsx:145]: It feels like this conversion should be a function. Not a blocker for this PR.

**Review by natachadelarosa — DISMISSED** [review]
LGTM

File: client/src/Operator/components/OperatorStake.tsx [hunk 2]
```diff
@@ -135,12 +138,12 @@ const OperatorStake = () => {
         return setFormError('We are not able to connect to the blockchain')
 
       try {
-        const block = await api.rpc.chain.getBlock()
-        const hash = await api.tx.domains
+        const block = await api[selectedChain.urls.page].rpc.chain.getBlock()
+        const hash = await api[selectedChain.urls.page].tx.domains
           .registerOperator(values.domainId, values.amountToStake * 10 ** tokenDecimals, {
```
Comment (jfrank-summit) [inline client/src/Operator/components/OperatorStake.tsx]: In testing there is a failure to decode amount:
```
Call: failed decoding domains.registerOperator:: Struct: failed on args: {"domain_id":"u32","amount":"u128","config":"{\"signingKey\":\"SpDomainsAppPublic\",\"minimumNominatorStake\":\"u128\",\"nominationTax\":\"Percent\"}"}:: Struct: failed on amount: u128:: Number needs to be an integer <= Number.MAX_SAFE_INTEGER, i.e. 2 ^ 53 - 1
```

Comment (jfrank-summit) [reply client/src/Operator/components/OperatorStake.tsx]: Never mind, this is working now 🤔

**Review by jfrank-summit — APPROVED** [review]
Thanks for getting this so fast!

**Pull request merged**